crates/youki/Cargo.toml

```diff
@@ -49,6 +49,7 @@ wasi-common = { version = "24.0.0", optional = true }
 tracing = { version = "0.1.40", features = ["attributes"] }
 tracing-subscriber = { version = "0.3.18", features = ["json", "env-filter"] }
 tracing-journald = "0.3.0"
+oci-spec = { version = "0.7.0", features = ["runtime"] }
```

Comment:
I guess I need to wait for https://github.com/containers/youki/issues/2920 and use `libcontainer:oci_spec` instead of `oci_spec`. I'd like to hear opinions from maintainers.

Reply:
In general I don't think there would be any problems with that, maybe even preferable to use `libcontainer::oci_spec`. Can you tell me what particular issue is with using `oci_spec` directly?

Reply:
Hey, the PR is merged annd oci-spec is updated to 0.8.0. You should probably rebase this branch on main.

Reply:
OK, I’ll update the dependency and use the `libcontainer::oci_spec` instead.

Reply:
fixed in 96a4557